ctms/bin/acoustic_sync.py

```diff
@@ -1,13 +1,46 @@
 #!/usr/bin/env python3
 """Run continuously in the background, syncing acoustic with our db."""
+import argparse
+import logging
 from time import monotonic, sleep
 
 from ctms import config
 from ctms.database import get_db_engine
 from ctms.sync import CTMSToAcousticSync
 
+LOGGER = None
+
+
+def _setup_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Should this be a new command line argument, or should it be from ``config_settings.logging_level``? That centralizes log config in ``CTMS_LOGGING_LEVEL``, but that has a default of ``INFO``.

Reply:
> Thanks @bsieber-mozilla! I don't have the credentials to try this out, but I was able to get the first log entry:
> 
> ```
> DEBUG:__main__:Begin Acoustic Sync Script.
> ```
> 
> I think you've figured out why the logs weren't printing. I think many, including these new ones, are correct at the DEBUG level.

Great callouts; I updated to use the Settings instead of through the cmdline. I opted to overload the property to modify the default to be debug (maybe I should leave it without overloading and just toggle the env var in the deploy?).

I also opted to use the configure logging helper method. I missed that originally thanks for setting that up.

```
LogLevel=DEBUG
{"Timestamp": 1621373414206429952, "Type": "ctms.sync", "Logger": "ctms", "Hostname": "bsieber-nvjgh5", "EnvVersion": "2.0", "Severity": 7, "Pid": 16554, "Fields": {"msg": "START: sync.sync_records"}}
{"Timestamp": 1621373414257826048, "Type": "ctms.sync", "Logger": "ctms", "Hostname": "bsieber-nvjgh5", "EnvVersion": "2.0", "Severity": 7, "Pid": 16554, "Fields": {"msg": "END: sync.sync_records"}}
```

Reply:
I wonder if I shouldn't have overloaded for a lower level default. After pushing I thought it could just be done on deploy. _Although may require an sre approval?_